packages/fxa-settings/src/components/DropDownAvatarMenu/index.test.tsx

```diff
@@ -107,12 +107,13 @@ describe('DropDownAvatarMenu', () => {
         </div>
       </AppContext.Provider>
     );
-    const dropDown = screen.queryByTestId('drop-down-avatar-menu');
+
+    const dropDownId = 'drop-down-avatar-menu';
```

Comment:
Hhhmm... Sorry, just thinking out loud here. Now we're defining the constant in three places (L54, L92, and L111).
What if we shift it up from each `it()` test scope into the parent `describe()` suite on L38.5 so we only need to define it once?